src/lib/crypto/snarky_tests/snarky_tests.ml

```diff
@@ -434,6 +434,44 @@ let api_tests =
   ; ("compile monadic API", `Quick, MonadicAPI.get_hash_of_circuit)
   ]
 
+module Protocol_circuits = struct
+  let proof_level, constraint_constants =
+    Genesis_constants.(Proof_level.Full, Constraint_constants.compiled)
```

Comment:
maybe a comment on why this configuration?

Reply:
done.